definitions/mlops-openai_application/dashboard.json

```diff
@@ -0,0 +1,341 @@
+                }
+              ],
+              "nullValues": {
+                "nullValue": "preserve"
+              },
+              "platformOptions": {
+                "ignoreTimeRange": false
+              },
+              "yAxisLeft": {
+                "zero": true
+              }
+            }
+          },
+          {
+            "title": "",
+            "layout": {
+              "column": 1,
+              "row": 10,
+              "width": 12,
+              "height": 1
+            },
+            "linkedEntityGuids": null,
+            "visualization": {
+              "id": "viz.markdown"
+            },
+            "rawConfiguration": {
+              "text": "## Completion samples"
+            }
+          },
+          {
+            "title": "",
+            "layout": {
+              "column": 1,
+              "row": 11,
+              "width": 12,
+              "height": 2
+            },
+            "linkedEntityGuids": null,
+            "visualization": {
+              "id": "viz.table"
+            },
+            "rawConfiguration": {
+              "facet": {
+                "showOtherSeries": false
+              },
+              "nrqlQueries": [
+                {
+                  "accountIds": [
+                    3734226
+                  ],
+                  "query": "FROM OpenAICompletion SELECT prompt, choices.0.text WHERE prompt is not NUll"
+                }
+              ],
+              "platformOptions": {
+                "ignoreTimeRange": false
+              }
+            }
+          },
+          {
+            "title": "",
+            "layout": {
+              "column": 1,
+              "row": 13,
+              "width": 12,
+              "height": 1
+            },
+            "linkedEntityGuids": null,
+            "visualization": {
+              "id": "viz.markdown"
+            },
+            "rawConfiguration": {
+              "text": "## ChatCompletion samples"
+            }
+          },
+          {
+            "title": "",
+            "layout": {
+              "column": 1,
+              "row": 14,
+              "width": 12,
+              "height": 2
+            },
+            "linkedEntityGuids": null,
+            "visualization": {
+              "id": "viz.table"
+            },
+            "rawConfiguration": {
+              "facet": {
+                "showOtherSeries": false
+              },
+              "nrqlQueries": [
+                {
+                  "accountIds": [
+                    3734226
+                  ],
+                  "query": "FROM OpenAICompletion SELECT messages, choices.0.message.content WHERE messages IS NOT NULL"
+                }
+              ],
+              "platformOptions": {
+                "ignoreTimeRange": false
+              }
+            }
+          }
+        ]
+      }
+    ],
+    "variables": []
```

Comment:
Hey @otaviocarvalho.
I fixed the dashboard.json. I ran the validation script locally and it passed 🙌

Reply:
Found an issue on the last build, I wonder if it is related to the node version. I will try to dig into it too.

Reply:
https://github.com/newrelic/entity-definitions/actions/runs/4945821254/jobs/8844607484?pr=1056

Reply:
I says to run `sanitize-dashboards` which I just did and it did some changes to the dashboard.json file. I re-pushed the PR. Hopefully it will pass this time.